src/lgdo/types/histogram.py

```diff
@@ -221,6 +225,15 @@ def __init__(
             as binning.
         attrs
             a set of user attributes to be carried along with this LGDO.
+        ignore_hist_flowbins
```

Comment:
Why not just `flow=False`? Saves some typing...

Reply:
:+1: implemented